### ISSUE TITLE
Rounded Corners

### ISSUE THREAD

**Issue by digitalbuddha** [Flipboard/bottomsheet#203]
Hi folks, would you accept a pr contribution to allow rounded corners like the new material spec?

**Comment by markrietveld** [comment]
Yes. We don't currently do a lot of development on this repo, but we're open to targeted changes like that.
